packages/medusa/src/loaders/services.ts

```diff
@@ -32,4 +36,29 @@ export default ({ container, configModule, isTest }: Options): void => {
       })
     }
   })
+
+  const moduleResolutions = configModule?.moduleResolutions ?? {}
+  for (const [_, resolution] of Object.entries(moduleResolutions)) {
+    if (resolution.shouldResolve) {
+      try {
+        const loadedModule = await import(resolution.resolutionPath!)
+        const loadedService = loadedModule.service
```

Comment:
suggestion: Should we not expect the module to export an object with the expected props types (service, loader) instead of having to explain that does variable are expected and must be exported, we can provide a type of the expected object to be exported by the module. It would also avoid I think a lot of questions and typo
```
export default {
  service: ...
  loader: ...
}